assets/js/backLink.js

```diff
@@ -1,4 +1,4 @@
-const backLink = document.querySelector('.govuk-back-link')
+const backLink = document.querySelector('.js-back-link')
```

Comment:
Using the class `.js-back-link` as the selector is better, both here for the JS and next file for the css, because if we use the standard govuk style as the selector then the back links on the Induction journey will be affected, and we dont want to do that (in this PR)